### ISSUE TITLE
Transcription: speaker definition

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#1085]
## Bug Report

**Problematic behavior**
If the transcription is requested, it is provided by email and through a Docs link.

I do get the transcribed content, but I don't have the identification of the meeting participants. It shows Speaker_01, Speaker_02, etc…

**Expected behavior/code**
When I receive a written transcription, I would like my participants' statements to be identified so I know who is saying what.

**Possible Solution**
It should be possible, initially, to have the list of people identified in the meeting in list form before the transcript.

And finally, the display of people who identified themselves and who spoke during the meeting, in the form of [First Name LAST NAME: statement]

**Additional context/Screenshots**

<img width="132" height="202" alt="Image" src="https://github.com/user-attachments/assets/85513045-dfbb-407e-b5de-4fe24dc42f0c" />

**Comment by lebaudantoine** [comment]
Ongoing work by @cameledev

**Comment by lebaudantoine** [comment]
duplicated with #486
